README.md

```diff
@@ -106,7 +107,7 @@ To build Ceph, follow this procedure:
    > Using the `-j` option with an argument appropriate to the hardware on
    > which the `ninja` command is run is expected to result in a successful
    > build. For example, to limit the job number to 3, run the command `ninja
-   > -j 3`. On average, each `ninja` job run in parallel needs approximately
+   > -j3`. On average, each `ninja` job run in parallel needs approximately
```

Comment:
"On average" and "approximately" are redundant.  I would `s/On average, e /E/`

Reply:
All this stuff is going to be rewritten (in a future PR). Your anti-pleonasm suggestion will not be forgotten when the time comes.

Reply:
Pleonasms are things I don't tolerate, condone, or up with which I will put ;)